exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,6 @@
 const name = " Daniel  ";
+const addTrim = name.trim();
+const newNameLength = addTrim.length;
```

Comment:
Be careful with your variable names. Try to make them as descriptive of the contents as you can. For example I'd personally call this 'trimmedName' instead of 'addTrim'.  To add a trim is a verb whereas the variable contents should describe the result of adding the trim. 'addTrim' would make more sense as a function name. i.e we call the function to 'add a trim' to a string.